SEC/Securities/SecuritiesListings.rdf

```diff
@@ -235,11 +232,39 @@
 				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-sec-sec-iss;isIssuedInForm"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-lst;RegisteredSecurityForm"/>
```

Comment:
how could you have an instance of RegisteredSecurityForm - it should be an individual ( required to avoid punning - see earlier comment)

Reply:
fixed - see above